lib/pages/navigation_page.dart

```diff
@@ -79,24 +83,33 @@ class _NavigationPageState extends State<NavigationPage> {
               child: Column(
                 mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                 children: [
-                  Text("To: " + _destination,
+                  FittedBox(
+                    child: Container(
+                      width: deviceWidth * 0.65,
```

Comment:
Could you factor out the float values like 0.65 into a variable so there are fewer magic values?